config/templates/cluster.yaml

```diff
@@ -165,8 +165,12 @@ kmsKeyArn: "{{.KMSKeyARN}}"
 # experimental:
 #   nodeDrainer:
 #     enabled: true
+#   awsEnvironment:
+#     enabled: true
+#     environment:
+#       CFNSTACK: '{ "Ref" : "AWS::StackId" }'
```

Comment:
Thanks for the explanation 😃 
I'm afraid that the syntax checking thing is too much for now.
I'd rather merge this as is and want to see how our users play with this!

> Isn't all of this advanced? :)

Certainly! 😆